src/golang/cmd/server/server/cron.go

```diff
@@ -44,12 +54,77 @@ func (s *AqServer) triggerMissedCronJobs(
 	}
 }
 
-// RunMissedCronJobs first gets the latest workflow run timestamp of all deployed workflows that are
+// backfillKilledJobs backfills all pending and running op/artf/DAG _results
+// and mark them as canceled.
+func (s *AqServer) backfillKilledJobs(ctx context.Context) error {
```

Comment:
How does this work for non-Aqueduct orchestrated workflows, such as Airflow. In that case we don't want to turn running operators and artifacts to cancelled.

Reply:
Interesting..how does the sync with airflow works? I think we can either bypass this backfill for these engines or runs the sync as a part of this method. I'd prefer to do the second option if there's an easy way to do so

Reply:
But as a first cut you can also just bypass Airflow DAGs for now, since that might be easiest to get you unblocked. This is more of an optimization anyways, as we would sync with Airflow anytime the user goes to the UI.